src/FakeItEasy/FakeItEasy.csproj

```diff
@@ -280,7 +280,10 @@
   <ItemGroup />
   <ItemGroup>
     <EmbeddedResource Include="Creation\CastleDynamicProxy\DynamicProxyResources.resx" />
-    <EmbeddedResource Include="ExceptionMessages.resx" />
+    <EmbeddedResource Include="ExceptionMessages.resx">
+      <Generator>PublicResXFileCodeGenerator</Generator>
+      <LastGenOutput>ExceptionMessages1.Designer.cs</LastGenOutput>
+    </EmbeddedResource>

```

Comment:
> (not sure how it got unlinked)

Apparently it was changed in https://github.com/FakeItEasy/FakeItEasy/commit/46becf4a6cc8d72ddefc8ca024167d29979efdde#diff-ad5947a74d7c146b709f60c87bad9886, and we missed it, which isn't really surprising given the size of the commit...